java/src/main/java/com/azure/graph/bulk/impl/impl/CosmosDBSQLBulkExecutor.java

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.graph.bulk.impl.impl;
+
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.azure.cosmos.models.CosmosBulkOperationResponse;
+import com.azure.cosmos.models.CosmosBulkOperations;
+import com.azure.cosmos.models.CosmosItemOperation;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.graph.bulk.impl.GraphBulkExecutor;
+import com.azure.graph.bulk.impl.ObjectToEdge;
+import com.azure.graph.bulk.impl.ObjectToVertex;
+import com.azure.graph.bulk.impl.model.GremlinEdge;
+import com.azure.graph.bulk.impl.model.GremlinVertex;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import reactor.core.publisher.Flux;
+
+import java.util.ArrayList;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+@Slf4j
+public class CosmosDBSQLBulkExecutor<V, E> implements GraphBulkExecutor<V, E> {
```

Comment:
This should be named similar to `GraphBulkExecutor`. Given that both are already bound to `Cosmos` SDK implementation perhaps use `Standard` or `Default` prefix? 

Alternatively, perhaps this should be internal and add a `GraphBulk` static class with a factory method.

eg. `GraphBulk.createBulkExecutor(container, mapper)`

Reply:
Can also consider a builder pattern since that is more the norm for java APIs.

Reply:
went with the builder pattern, it's being used quite a bit within the solution already.